portfolio/src/main/webapp/slideshow.css

```diff
@@ -0,0 +1,124 @@
+:root {
+  --blue-default: #25274d;
+  --blue-secondary: #29648a;
+  --blue-active: #2E9cca;
+  --text-color: #fff;
+}
+
+/* 
+Position the slide container so that
+we can use the left-right arrows. 
+*/
+.slide-container {
+  position: relative;
+}
+
+/* 
+By default, we don't want to display any slides, 
+since we only want one at a time. 
+*/
+.slide {
+  display: none;
+  border-bottom: none;
+  animation: fade-in ease 3s;
+  -webkit-animation: fade-in ease 3s;
+}
+
+/* 
+We want to style the previous and 
+next buttons identically except for the 
+fact that one is on the :right
+and the other is on the left. 
+*/
+.previous-button, .next-button {
+  cursor: pointer;
+  position: absolute;
+  top: 40%;
+  width: auto;
+  padding: 16px;
+  margin-top: -50px;
+  color: var(--text-color);
+  font: bold 100%/1.6;
+  border-radius: 0 3px 3px 0;
+  user-select: none;
+  -webkit-user-select: none;
+}
+
+/* Next button goes on the right. */
+.next-button {
+  right: 0;
+  border-radius: 3px 0 0 3px;
+}
+
+/* Hover behaviour for previous and next buttons. */
+.previous-button:hover, .next-button:hover {
+  background-color: var(--blue-default);
+  opacity: 0.8;
+}
+
+/* Styling for image captions. */
+.slide-caption {
+  margin-top: none;
+  text-align: center;
+  background-color: var(--blue-default);
+  padding: 2px 16px;
+  color: var(--text-color);
+  font: bold;
+}
+
+/* 
+This 'row' will hold all of the 'columns' containing 
+each image in the slideshow. 
+*/
+.slideshow-row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+/* 
+There are six images within my specific slide show, how
+would I change this css to fit any size n slideshow? 
+*/
```

Comment:
Make a TODO and bug to figure this out.

```
TODO(#XX): figure out to to make this not depend on the number of images.
```

Reply:
Will make a TODO, for now I'll create an issue and go back to it if I have time, but I think I'm going to start on the week 3 project as soon as possible.